packages/core/src/client.test.ts

```diff
@@ -1149,7 +1149,7 @@ describe('Client', () => {
   describe('FHIRcast', () => {
```

Comment:
Would it be ok to move these tests into a separate file?  It's a pretty clean cut on functionality, and would help get this file under control.

Not urgent, we can do that in separate PR.